app/livechat/server/methods/requestScreenshare.js

```diff
@@ -0,0 +1,20 @@
+import { Meteor } from 'meteor/meteor';
+
+import { Messages } from '../../../models';
+
+Meteor.methods({
+	async 'livechat:requestScreenshare'(roomId) {
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-not-authorized', 'Not authorized', { method: 'livechat:requestScreenshare' });
+		}
+
+		const guest = Meteor.user();
+
+		Messages.createWithTypeRoomIdMessageAndUser('request_screenshare_access', roomId, '', guest, {
```

Comment:
This message type will be sent only when the provider calls it.
So, we'll need to implement a `ScreenSharingManager` to handle that kind of request instead of calling this message here.